### DISCUSSION TITLE
rerun ml jobs when change ML details?

### DISCUSSION THREAD

**Discussion by titro** [immich-app/immich#11827] [Q&A]
Hi Folks,

i switched from CPU based ML to Openvino ML with 1.111 release and since then the search is only showing bulls* with context search.

Do i have to rerun the ml job ? and what if i change the ml model due to #11468 ? Also?

Thanks!

**Comment by imohitkr** [comment]
https://immich.app/docs/features/smart-search#CLIP-model 

Its mentioned in the website that you need to re-run for all assets.
